### PR TITLE
EMP mines

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Adjusted EMP mine behavior

## Why It's Good For The Game

This makes mines less obnoxious to deal with.

## Changelog
```changelog
fix: flash radius on explosions now does something!
tweak: The EMP landmine now only detonates if a mob walks on it, or it is touched
tweak: EMP mines can now be disabled by a multitool, like regular landmines.
tweak: picking up an active EMP mine triggers and disables it.
fix: De-russified some strings on regular landmines.
```

### CODE REVIEW

**Review by zackerycodestoo — CHANGES_REQUESTED** [review]
As Cebutris said in coding discussions, could this be split into two pulls, one for mines and one for snacks? I know I've been a bit lenient about unrelated changes in the past

File: code/game/objects/items/weapons/mine.dm [hunk 1]
```diff
@@ -71,7 +71,7 @@
 				)
 		if (do_after(user, 5))
 			user.visible_message(
-				SPAN_DANGER("[user] attempts to pick up the [src] only to hear a beep as it explodes in your hands!"),
+				SPAN_DANGER("[user] attempts to pick up the [src] only to hear a beep as it explodes in thier hands!"),//Occulus Edit: Russian grammar
```
Comment (Michiyamenotehifunana) [inline code/game/objects/items/weapons/mine.dm:74]: thier

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
typo

**Review by Michiyamenotehifunana — APPROVED** [review]
Good to go

**Pull request merged**